src/components/login/SSOLogin.tsx

```diff
@@ -320,6 +347,51 @@ export default class SSOLogin extends Component<SSOLoginProps, SSOLoginState> {
         this.setState({ configMap: value })
     }
 
+    handleOnBlur(): void {
+        if (this.state.configMap !== SwitchItemValues.Configuration) return
```

Comment:
can we try to create some common method to reduce the duplicacy between handleOnBlur and handleOnFocus?